admin_manual/installation/nginx_configuration.rst

```diff
@@ -73,11 +73,23 @@ Nginx Configuration
       deny all;
       }
 
+    location ~* ^/(build|tests|config|lib|3rdparty|templates)/.* {
+      deny all;
+      }
+
+    location ~* ^/(autotest|occ|issue|indie|db_|console).* {
+      deny all;
+      }
+
     location / {
      # The following 2 rules are only needed with webfinger
      rewrite ^/.well-known/host-meta /public.php?service=host-meta last;
      rewrite ^/.well-known/host-meta.json /public.php?service=host-meta-json last;
 
+     rewrite ^/apps/calendar/caldav.php /remote.php/caldav/ last;
+     rewrite ^/apps/contacts/carddav.php /remote.php/carddav/ last;
```

Comment:
Personally, I think these two rewrites should be removed from the `.htaccess`, not added here